### ISSUE TITLE
Link submitter name on entry to related profile when available

### ISSUE THREAD

**Issue by alanmoo** [mozilla/network-pulse#625]

**Comment by xmatthewx** [comment]
@mmmavis this is now unblocked. woohoo! 

I guess we should keep in mind that some authors might not have a profile with legacy entries.

**Comment by mmmavis** [comment]
@xmatthewx  by "author" do we mean the entry submitter?

**Comment by xmatthewx** [comment]
Yep!

**Comment by xmatthewx** [comment]
merged into feature branch
